lodash.js

```diff
@@ -3699,8 +3699,13 @@
       return function(number, precision) {
         precision = precision ? toInteger(precision) : 0;
         if (precision) {
-          precision = pow(10, precision);
-          return func(number * precision) / precision;
+          // Shift decimal point with exponential operations
+          number = (number + 'e').split('e');
```

Comment:
`number + 'e'` should probably become `+number + 'e'`, otherwise this change will start accepting invalid input that looks like valid input followed by "e" (e.g., `"5e"` or `"7e55e…"`—solid test cases, FYI :wink:).